packages/cms-lib/hubdb.js

```diff
@@ -143,6 +147,14 @@ function convertToJSON(table, rows) {
 async function downloadHubDbTable(portalId, tableId, dest) {
   const table = await fetchTable(portalId, tableId);
 
+  dest = path.resolve(dest || `${table.name}.hubdb.json`);
```

Comment:
Where is `getCwd()` called to resolve the fallback relative to `getCwd()`?

Reply:
Ah, thanks. I thought that `path.resolve()` was enough but I see that `getCwd()` takes care of getting the correct cwd now.